wain-syntax-text/src/wat2wasm.rs

```diff
@@ -265,6 +302,8 @@ pub fn wat2wasm<'s>(
         local_indices: Indices::new(),
         next_local_idx: 0,
         label_stack: LabelStack::new(source),
+        tentatives: std::mem::replace(&mut parsed.module.tentatives, Vec::new()),
```

Comment:
```suggestion
        tentatives: std::mem::take(&mut parsed.module.tentatives),
```

意味は同じですが，`take` でよりシンプルになりそうです